src/components/new-safe/create/steps/AdvancedOptionsStep/index.tsx

```diff
@@ -0,0 +1,220 @@
+                arrow
+                placement="top"
+              >
+                <span style={{ display: 'flex' }}>
+                  <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
+                </span>
+              </Tooltip>
+            </Typography>
+            <Typography variant="body2" mb={2}>
+              Changes the used master copy and fallback handler of the Safe.
+            </Typography>
+            <Controller
+              control={control}
+              name="safeVersion"
+              render={({ field }) => (
+                <TextField select {...field} label="Safe version">
+                  <MenuItem value="1.4.1">1.4.1 (latest)</MenuItem>
+                  <MenuItem value="1.3.0">1.3.0</MenuItem>
+                </TextField>
+              )}
+            />
+          </Box>
+
+          <Divider />
+          <Box className={layoutCss.row}>
+            <Typography variant="h4" fontWeight={700} display="inline-flex" alignItems="center" gap={1}>
+              Salt nonce
+              <Tooltip
+                title="The salt nonce changes the predicted Safe address. It can be used to re-create a Safe from anothet chain or to create a specifcic Safe address"
+                arrow
+                placement="top"
+              >
+                <span style={{ display: 'flex' }}>
+                  <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
+                </span>
+              </Tooltip>
+            </Typography>
+            <Typography variant="body2" mb={2}>
+              Impacts the derived Safe address
+            </Typography>
+            <NumberField
+              {...register(AdvancedOptionsFields.saltNonce, {
+                validate: async (value) => {
+                  if (isNaN(value)) {
+                    return 'Salt nonce must be a number'
+                  }
+                  if (value < 0) {
+                    return 'Salt nonce must be positive'
+                  }
+                },
+              })}
+              label="Salt nonce"
+              error={Boolean(formState.errors[AdvancedOptionsFields.saltNonce]) || Boolean(isDeployed)}
+              helperText={
+                formState.errors[AdvancedOptionsFields.saltNonce]?.message ?? Boolean(isDeployed)
+                  ? 'The Safe is already deployed. Use a different salt nonce.'
+                  : undefined
+              }
+            />
+          </Box>
+          <Grid container spacing={3}>
+            <ReviewRow
+              name="New Safe address"
+              value={
+                predictedSafeAddress ? (
+                  <EthHashInfo address={predictedSafeAddress} hasExplorer showCopyButton />
+                ) : (
+                  <Skeleton />
+                )
+              }
+            />
+          </Grid>
```

Comment:
I would suggest wrapping this in a `<Box className={layoutCss.row}>` as well so that it is aligned with the other rows.
<img width="612" alt="Screenshot 2024-08-05 at 11 18 52" src="https://github.com/user-attachments/assets/acd87cbd-4e05-4232-966b-480a53c7f4b3">